plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -327,6 +327,7 @@ func (w *windowsEventLog) getRecord(evtHandle EvtHandle) (*windowsEventLogRecord
 	if err != nil {
 		return nil, fmt.Errorf("UTF16ToUTF8Bytes() err %v", err)
 	}
+
```

Comment:
Can the formatting changes come in a diff pr?

Reply:
It's only a handful of formatting changes and they're all purely cosmetic. I think they're fine to keep in here.